corona-warn-app/cwa-app-android
# Deletion of vaccination certificates (EXPOSUREAPP-7111)

### Description
This PR implements deletion of Vaccinations in the Vaccination List View via
- overflow menu
![image](https://user-images.githubusercontent.com/10398034/118668289-042f2080-b7f5-11eb-90b3-4dc112fb7ac5.png)

- swipe to delete
![image](https://user-images.githubusercontent.com/10398034/118668327-0db88880-b7f5-11eb-95d3-41e927108727.png)


In case the last vaccination is deleted, the app navigates back to the HomeScreen. 

### How to test
- register some vaccinations
- try the overflow menu and swipe to delete
- See if vaccinations are properly deleted, if the last vaccination is deleted, you should be at the home screen again.
